.devenv

```diff
@@ -1,6 +1,6 @@
 NAME="timeoverflow"
 DISTRIBUTION="ubuntu"
-RELEASE="xenial"
+RELEASE="bionic"
 ARCH="amd64"
 HOST="local.$NAME.org"
 PROJECT_NAME="timeoverflow"
```

Comment:
`PYTHON_INTERPRETER="python2.7"`

Can we use Python 3, now?

Reply:
Yes, we can start to use the interpreter of Python 3.

Remember that to use the Python 3 we need to change the `ansible_python_interpreter` set in the inventory group vars: 
https://github.com/coopdevs/timeoverflow-provisioning/blob/master/inventory/group_vars/all.yml#L2